scope

```diff
@@ -99,7 +99,7 @@ weave_dns_present() {
 }
 
 set_docker_bridge_ip() {
-    DOCKER_BRIDGE_IP=$(ip -4 addr show dev $DOCKER_BRIDGE | grep -m1 -o 'inet [.0-9]*')
+    DOCKER_BRIDGE_IP=$(docker run --rm --net=host gliderlabs/alpine ip -f inet address show dev $DOCKER_BRIDGE | grep -m1 -o 'inet \([.0-9]\)*')
```

Comment:
Can you use the weavescope image with --entrypoint=/bin/sh please?

That way we'll automatically be labelled with works.weave.role=system and filtered when that happens.